gradle.properties

```
@@ -1 +1 @@
-release_version = 2.7.0
\ No newline at end of file
+release_version = 2.7.1
```

```suggestion
release_version = 2.8.0
```
Update the minor version because the minor version of the dependency is updated